.docs/shared-responsabilities.md

```diff
@@ -0,0 +1,61 @@
+# Understanding your responsibilities when using Landing Zone Deployable Architecture
+
+In IBM Cloud, the responsibilities for deploying, operating, and securing products are shared between IBM and you, the application provider. It is important for you to understand these responsibilities for every IBM Cloud service that is deployed as part of this Deployable Architecture (DA).
+
+This document walks through the main categories outlined in (https://cloud.ibm.com/docs/overview?topic=overview-shared-responsibilities)[Shared responsibilities for using IBM Cloud products] for the specific case of the Landing Zone Deployable Architecture. 
+
+This document augments and completes (but does not replaces) the documentation on shared responsabilities for each of the provisioned services with specifics to the DA - see [framework-financial-services-shared-responsibilities](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-shared-responsibilities).
+
+The scope of this document only includes the capabilities surfaced in IBM Cloud Catalog, and does not cover capabilities such as Teleport, Big IP F5 support that are specific to the open-source version of this DA.
+
+
+## Incident and operations management
+
+Incident and operations management includes tasks such as monitoring, event management, high availability, problem determination, recovery, and full state backup and recovery.
+
+The DA does not provide further support in this area. Incident and operations management tasks are outlines for each provisioned services at https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-shared-responsibilities

```

Comment:
outlines -> outlined..

For the links you put it within the sentence -
Incident and operations management tasks are outlined for [each provisioned services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-shared-responsibilities)